src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/QueryExecutor.java

```diff
@@ -93,6 +144,23 @@ public Uni<QueryOuterClass.ResultSet> executeSchemaChange(QueryOuterClass.Query
         QueryOuterClass.Query.newBuilder(query).setParameters(params).buildPartial());
   }
 
+  /**
```

Comment:
we can start remove the bridge methods in QueryExecutor
since they are not used anymore

Reply:
Done